gems/sorbet/test/hidden-method-finder/hidden_methods.bzl

```diff
@@ -33,7 +33,7 @@ def _hidden_methods_test(test_path, ruby):
     native.genrule(
         name = "actual_{}/{}".format(ruby, test_path),
         message = "Running {} ({})".format(test_path, ruby),
-        tools = [
+        exec_tools = [
```

Comment:
Do we believe this is actually benefitical.
AFAIK it currently creates one more build configuration and thus is actually harmful.

Reply:
As far as I can tell, doing it this way cuts down to just one ruby build for each version, instead of two. I was seeing one for execution and one for host before, and now I only see one for execution.

Reply:
Resolved via DM, I didn't realize that `exec_tools` would add a third configuration, not just skip the host configuration.